packages/theme/src/components/Difficulty.tsx

```diff
@@ -1,8 +1,8 @@
-import { useI18n } from '@/i18n'
+import { useI18n } from '@edgeandnode/components'
```

Comment:
this function should be generic (to accept different translations) since now `useI18n` lives in `packages/theme` (or maybe I should rename it `packages/nextra-theme`)

A current function definition is

```ts
export const useI18n = () => _useI18n<typeof translations>()
```

Reply:
Oh, right, each package that uses the theme might have its own translatable strings... still, the theme itself will only ever refer to strings that exist within the theme, so it's almost like we'd need 3 layers of this. Oh well, I guess this works, it just doesn't provide the nice autocomplete experience. :P

Reply:
yeah, we lose autocomplete